src/components/inputs/Switch.tsx

```diff
@@ -0,0 +1,33 @@
+import * as RadixSwitch from '@radix-ui/react-switch';
+import React from 'react';
+
+const Switch = ({
+  checked,
+  onChange,
+  className,
+}: {
+  checked: boolean;
+  onChange: (checked: boolean) => void;
+  className?: {
+    root?: string;
+    thumb?: string;
+  };
+}) => (
+  <RadixSwitch.Root
+    onCheckedChange={onChange}
+    checked={checked}
+    className={
+      className?.root ??
+      'relative h-[25px] w-[42px] cursor-default rounded-full bg-blackA6 shadow-[0_2px_10px] shadow-blackA4 outline-none focus:shadow-[0_0_0_2px] focus:shadow-black data-[state=checked]:bg-black'
```

Comment:
Would be good to use rem's here too.

Reply:
[fb6ff4c](https://github.com/ar-io/arns-react/pull/639/commits/fb6ff4cb2ca616122b7253290564f3c2f3f54533)